src/components/views/settings/SetIdServer.js

```diff
@@ -32,14 +32,14 @@ import {SERVICE_TYPES} from "matrix-js-sdk";
  * @param {string} u The url to be abbreviated
  * @returns {string} The abbreviated url
  */
-function abbreviateUrl(u) {
+export function abbreviateIdentityUrl(u) {
```

Comment:
We should probably move this into a utility function file and import that rather than exporting it from a component file.